typedmodels/models.py

```diff
@@ -201,6 +194,27 @@ def get_types(subcls):
         return cls
 
     @staticmethod
+    def _get_model_manager(cls):
+        # set default manager. this will be inherited by subclasses, since they are proxy models
+        manager = None
+
+        def get_manager(subcls):
+            class Manager(TypedModelManager, subcls.__class__):
+                pass
+            return Manager
+
+        if not cls._default_manager:
```

Comment:
This looks a lot like the old implementation to me. Can you explain what the difference is?